code/midtown/mmcity/cullcity.cpp

```diff
@@ -85,3 +98,162 @@ void mmCullCity::Cull()
+
+    Loader()->BeginTask(LOC_STRING(MM_IDS_LOADING_CITY_LAYOUT), 0.0f);
+
+    DLPTemplate* city_dlp = nullptr;
+    DLPTemplate* lm_dlp = nullptr;
+
+    if (DevelopmentMode)
+    {
+        city_dlp = GetDLPTemplate(city_path);
+        if (!city_dlp)
+            Quitf("Unable to load city '%s'", name);
+
+        lm_dlp = GetDLPTemplate(lm_path);
+    }
+
+    BeginMemStat("mmCullCity::Init");
+    InitProblems();
+
+    Camera = camera;
+    asNode::AddChild(&BangerDataManager);
+    asNode::AddChild(&BangerActiveManager);
+    asNode::AddChild(&BangerManager);
+    asNode::AddChild(&asnode34AF0);
+    asNode::AddChild(&PHYS);
+    asNode::AddChild(&RenderWeb);
+    asNode::AddChild(&Particles);
+
+    agiMeshCardVertex* vertex = NULL;
+
+    if (MMSTATE.Weather == mmWeather::Rain)
+        vertex = RainMeshCard;
+    else
+        vertex = agiMeshSet::DefaultQuad;
+
+    Particles.Init(100, 4, 4, 4, vertex);
+
+    SnowBirthRule.SetName("SnowRule");
+    AddChild(&SnowBirthRule);
+    SnowBirthRule.Load();
+
+    RainBirthRule.SetName("RainRule");
+    AddChild(&RainBirthRule);
+    RainBirthRule.Load();
+
+    BirthRule = 0;
+
+    Sky.Init("mmsky"_xconst);
+    Loader()->EndTask(0.0f);
+
+    BeginMemStat("mmCullCity.WEB");
+    RenderWeb.Load(name, 1);
+    EndMemStat();
+
+    HitIdBound = RenderWeb.HitIdBound;
+    HitIdBound->AddRef();
+
+    BuildingChain.Init(RenderWeb.MaxCells);
+    ObjectsChain.Init(RenderWeb.MaxCells);
+    ShadowChain.Init(RenderWeb.MaxCells);
+
+    InitObjectDetail();
+
+    LastInstance = (mmInstance*) mmInstanceHeap.HeapHead;
+    ResetInst = new mmYInstance();
+
+    LoadBangers(name);
+
+    if (MMSTATE.GameMode == mmGameMode::Blitz)
+    {
+        LoadBangers(arts_formatf<64>("%s_b%d", name, MMSTATE.EventId));
+    }
+    else if (MMSTATE.GameMode == mmGameMode::Checkpoint)
+    {
+        LoadBangers(arts_formatf<64>("%s_r%d", name, MMSTATE.EventId));
+    }
+    else if (MMSTATE.GameMode == mmGameMode::Circuit)
+    {
+        LoadBangers(arts_formatf<64>("%s_c%d", name, MMSTATE.EventId));
+    }
+    else if (MMSTATE.GameMode == mmGameMode::Cruise)
+    {
+        LoadBangers(arts_formatf<64>("%s_roam", name));
+    }
+    else if (MMSTATE.GameMode == mmGameMode::CnR)
+    {
+        LoadBangers(arts_formatf<64>("%s_cops", name));
+    }
+
+    ResetInst = new mmYInstance();
+
+    LoadFacades(name);
+
+    if (CHICAGO)
+    {
+        Vector3 pos_1 = {995.782, 0.173, 1188.804};
+        Vector3 pos_2 = {995.782, 0.173, 742.411};
+        Vector3 pos_3 = {1015.383, 0.173, 1188.805};
+        Vector3 pos_4 = {1015.383, 0.173, 742.411};
+
+        mmRunwayLight* light_1 = new mmRunwayLight("fxltglow"_xconst, pos_1, pos_2);
+        mmRunwayLight* light_2 = new mmRunwayLight("fxltglow"_xconst, pos_3, pos_4);
+        BuildingChain.Parent(light_1, 35);
+        BuildingChain.Parent(light_2, 35);
+
+        InitTimeOfDayAndWeather();
```

Comment:
This code and onwards shouldn't be inside `if (CHICAGO)`

Reply:
Hopefully the new code matches what you intended. 

In the future it would be nice to have something like this (next to the Chicago case)
if (CityName = "custom_city")
      ... custom Runway lights ...
      ... custom Runway lights ...